ruby/test/minitest/reporters/order_reporter_test.rb

```diff
@@ -13,7 +13,7 @@ def test_before_test
       @reporter.before_test(runnable('a'))
       @reporter.before_test(runnable('b'))
       @reporter.report
-      assert_equal ['Minitest::Test#a', 'Minitest::Test#b'], File.readlines(log_path).map(&:chomp)
+      assert_equal ['Minitest::Result#a', 'Minitest::Result#b'], File.readlines(log_path).map(&:chomp)
```

Comment:
This doesn't feel correct at all.

Reply:
There's some weird definitions going on in CI Queue:
https://github.com/Shopify/ci-queue/blob/master/ruby/lib/minitest/queue.rb#L119-L135

Reply:
This is to provide compatibility between different versions of Minitest. That is why we test 2 different versions on CI.